### ISSUE TITLE
Suggest import mp4

### ISSUE THREAD

**Issue by maikmller** [MrBrax/LiveStreamDVR#273]
Possibility to insert external mp4 files (mp4 that are in a folder) into the dashboard

**Comment by MrBrax** [comment]
how expanded? like a full on entry for each file? because they're pretty hardcoded to be streamers > vods right now. or just a file list where you can open/stream the files?

**Comment by maikmller** [comment]
I have old VODS on my server that I recorded with another program. And I would like to display this in the dashboard of the streamer.

**Comment by MrBrax** [comment]
![1660903516865](https://user-images.githubusercontent.com/1517911/185595982-bd9f541c-9ff2-4318-9e41-fb8b7c45f9ad.png)
something like this? it adds/deletes automatically when dropping videos in the channel folder

**Comment by maikmller** [comment]
exactly like that.

**Comment by MrBrax** [comment]
you can try it out if you switch to the develop branch/image, i'll push it to master in the future

**Comment by maikmller** [comment]
So far it works quite well. But is it intentional that the local files can not be played if you are not logged in?

**Comment by MrBrax** [comment]
nope, just a config value not being sent to guest clients. solved now

**Comment by CappiSteijns** [comment]
Could you explain more how this works? When adding .mp4 vods to the channel folder nothing changes. Added a test vod to Channel folder and saved_vods folder but it is now showing up on the webui. 

![image](https://user-images.githubusercontent.com/84481241/199500297-ee8df744-2b4e-4492-b857-a6b1a4a97cdc.png)

In this instance I added "It Takes Two" to: ../vods/vods/lekkerspelen and also to : ../vods/saved_vods/lekkerspelen

Logs dont show any info about this.

Also enabled "Enable local videos" in the settings.

**Comment by MrBrax** [comment]
that should be enough, but i could just guess that there's something with docker and folder watching that doesn't work properly

**Comment by kymograph** [comment]
I am having the same problem. For me, I am running Docker on Windows with WSL2 and this issue would be an explanation: https://github.com/docker/for-win/issues/8479 for the behaviour.

However, what about having a regular check on the file system and add found files upon that? It could run, e.g., once each day or once each hour, something like this? What do you think?
